src/components/slider/attributes.js

```diff
@@ -0,0 +1,221 @@
+            'value when an event of type `updateevent` is received. If',
+            'undefined, the data argument itself is used. If a string,',
+            'that property is used, and if a string with dots, e.g.',
+            '`item.0.label`, then `data[\'item\'][0][\'label\']` will',
+            'be used. If an array, it is matched to the respective',
+            'updateevent item or if there is no corresponding updatevalue',
+            'for a particular updateevent, it is interpreted as `undefined` and defaults to the data property itself.'
+        ].join(' ')
+    },
+
+    lenmode: {
+        valType: 'enumerated',
+        values: ['fraction', 'pixels'],
+        role: 'info',
+        dflt: 'fraction',
+        description: [
+            'Determines whether this color bar\'s length',
+            '(i.e. the measure in the color variation direction)',
+            'is set in units of plot *fraction* or in *pixels.',
+            'Use `len` to set the value.'
+        ].join(' ')
+    },
+    len: {
+        valType: 'number',
+        min: 0,
+        dflt: 1,
+        role: 'style',
+        description: [
+            'Sets the length of the color bar',
+            'This measure excludes the padding of both ends.',
+            'That is, the color bar length is this length minus the',
+            'padding on both ends.'
+        ].join(' ')
+    },
+    x: {
+        valType: 'number',
+        min: -2,
+        max: 3,
+        dflt: -0.05,
+        role: 'style',
+        description: 'Sets the x position (in normalized coordinates) of the slider.'
+    },
+    xpad: {
+        valType: 'number',
+        min: 0,
+        dflt: 10,
+        role: 'style',
+        description: 'Sets the amount of padding (in px) along the x direction'
+    },
+    ypad: {
+        valType: 'number',
+        min: 0,
+        dflt: 10,
+        role: 'style',
+        description: 'Sets the amount of padding (in px) along the x direction'
+    },
+    xanchor: {
+        valType: 'enumerated',
+        values: ['auto', 'left', 'center', 'right'],
+        dflt: 'left',
+        role: 'info',
+        description: [
+            'Sets the slider\'s horizontal position anchor.',
+            'This anchor binds the `x` position to the *left*, *center*',
+            'or *right* of the range selector.'
+        ].join(' ')
+    },
+    y: {
+        valType: 'number',
+        min: -2,
+        max: 3,
+        dflt: 1,
+        role: 'style',
+        description: 'Sets the y position (in normalized coordinates) of the slider.'
+    },
+    yanchor: {
+        valType: 'enumerated',
+        values: ['auto', 'top', 'middle', 'bottom'],
+        dflt: 'bottom',
+        role: 'info',
+        description: [
+            'Sets the slider\'s vertical position anchor',
+            'This anchor binds the `y` position to the *top*, *middle*',
+            'or *bottom* of the range selector.'
+        ].join(' ')
+    },
+
+    transition: {
```

Comment:
I kept this nested format so that it perfectly mirrors the way you'd define transition config for an animation. Otherwise it would need to be `transitionduration:` and `transitioneasing:`, which isn't bad. Open to thoughts here.